taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/CamundaProcessengineRequester.java

```diff
@@ -254,7 +264,7 @@ public boolean deleteProcessInstanceWithId(String processInstanceId, boolean ski
   public boolean isCorrectAssignee(String camundaTaskId, String assigneeValueToVerify) {
 
     String requestUrl = BASIC_ENGINE_PATH + this.processEngineKey + TASK_PATH + "/" + camundaTaskId;
-    HttpEntity<String> requestEntity = prepareEntityFromBody("{}");
+    HttpEntity<Void> requestEntity = httpHeaderProvider.prepareEntityFromBodyForCamundaRestApi();
```

Comment:
Why do you change the method from "prepareEntityFromBody" to "prepareEntityFromBodyForCamundaRestAPI" so that it uses camunda header here?

Reply:
Because I need also one for OutboxRestApi